spec/features/employee/administrator/user/administrator_edit_user_spec.rb

```diff
@@ -0,0 +1,39 @@
+feature = <<~HEREDOC
+  When administrator wants to edit user valid credentials,
+  he wants to edit just password of user by new and confirmationed password,
+  he wants to edit others credentials of user without entering a password,
+  so that user have beening  new credentials
```

Comment:
cannot understand this paragraph. Feature is that administrator wants to edit user so that user has new data, and that's all.